Mathlib/Algebra/Order/Ring/Abs.lean

```diff
@@ -157,7 +157,7 @@ private def geomSum : ℕ → α
   | n + 1 => a * geomSum n + b ^ (n + 1)
 
 private theorem abs_geomSum_le : |geomSum a b n| ≤ (n + 1) * max |a| |b| ^ n := by
-  induction' n with n ih; · simp [geomSum]
+  induction n with | zero => simp [geomSum] | succ n ih =>
```

Comment:
Was this formatting (`| zero => ... | succ n ih =>` on the same line) discussed on Zulip?

Reply:
https://leanprover.zulipchat.com/#narrow/channel/287929-mathlib4/topic/induction.20cases.20on.20the.20same.20line

Reply:
I've capped lines like this with `?_` per @kmill's [comment](https://leanprover.zulipchat.com/#narrow/channel/287929-mathlib4/topic/induction.20cases.20on.20the.20same.20line/near/506013193).